doc/ale-supported-languages-and-tools.txt

```diff
@@ -101,6 +101,7 @@ Notes:
   * `gcc` (`cc`)
   * `uncrustify`
 * Cairo
+  * `scarb`
```

Comment:
Mark the tool as requiring the file to be saved here and in Markdown. See other examples.

Reply:
Done!